cmd/clusterctl/client/repository/overrides.go

```diff
@@ -69,6 +70,11 @@ func (o *overrides) Path() string {
 	f, err := o.configVariablesClient.Get(overrideFolderKey)
 	if err == nil && strings.TrimSpace(f) != "" {
 		basepath = f
+
+		evaluatedBasePath, err := envsubst.Eval(basepath, os.Getenv)
+		if err == nil {
```

Comment:
@fabriziopandini We have a few alternatives here:
* log the error
* silently ignore the error
* change the interface so the Path() func can return an error 

What would you prefer?

Reply:
Given that we are backporting, let's log the error; the we can eventually made a followup PR to change the interface in main

Reply:
Makes sense, done ptal :)